attributes/network/cddl/3_kvstore.cddl

```diff
@@ -25,3 +23,23 @@ kvstore.get@return = {
     ? 0 => bstr,
 }
 ; end::get[]
+
+; tag::query[]
+kvstore.query@param = {
+    ; The key to query
+    0 => bstr,
+}
+
+kvstore.query@return = {
+    ; The key owner
+    0 => non-anonymous-identity,
```

Comment:
I think this should be optional. WDYT?

Reply:
Agreed.

Reply:
What I mean is that there might not be an owner.